odoo/addons/base/models/ir_attachment.py

```diff
@@ -247,7 +247,12 @@ def _compute_mimetype(self, values):
 
     def _check_contents(self, values):
         mimetype = values['mimetype'] = self._compute_mimetype(values)
-        xml_like = 'ht' in mimetype or 'xml' in mimetype # hta, html, xhtml, etc.
+        # test for suspicious, xml-like mime types to avoid certain XSS attacks
+        # (may be more appropriate to use a whitelist approach instead...)
```

Comment:
I'm not sure that creating a whitelist in a stable version is a good idea.
If you want to inform that maybe this can be made in a better way, you can create a feedback.

Reply:
How do I create the feedback?